portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,132 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>get to know Dez</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <img src="/desmondsimages/headerpic.png" 
+             alt="Dez Image #1" 
+             style="float:left;"
+             width="150px;"
+             height="150px;">
+        <h1>
+             get to know Dez
+        </h1>
+            <p class="leftpad1">
+                <strong>NAME</strong>: Desmond Talton <br><br>
+                <strong>COLLEGE</strong>: The Georgia Institute of Technology <br><br>
+                <strong>MAJOR</strong>: Computer Science <br><br>
+            </p>
+            <button onclick="funFactGenerator()">Random Fun Fact</button>
+            <div id="factgetter">
+                Press to Generate a Fun Fact.
+            </div>    
+        <h>
+            <br><br>
+            <img src="/desmondsimages/dezinsuit.jpg" 
+            alt="Dez Image #2" 
```

Comment:
Can you move the width and height of images to css file?

Reply:
`style="float:right;"` should also be moved to css, I might forgot mention. Please reply to this comment once you done.

Reply:
done.